app/views/devise/registrations/edit.html.erb

```diff
@@ -49,6 +49,11 @@
         <%= f.submit "Update User Account", class: 'btn btn-primary' %>
       </div>
     <% end %>
+    <h3 class="fw-light mt-5">Logout Account</h3>
+
+     <%= button_to destroy_user_session_path, class: 'btn btn-outline-primary', method: :delete do %>
+            <i class="fas fa-sign-out-alt"></i> Sign out
```

Comment:
Is it intended that 'Sign out' is _outside_ the italic-tag here? If so, the tag could be self-closing, right?

Reply:
It's a FontAwesome icon 😅